core/src/main/scala/com/salesforce/op/filters/AllFeatureInformation.scala

```diff
@@ -42,6 +42,7 @@ package com.salesforce.op.filters
  *                               1st level keys correspond to response keys
  *                               2nd level keys correspond to predictor keys with values being
  *                               null-label leakage corr. value
+ * @param avgtextLength          average length of text features
```

Comment:
hmm, I dont see `avgtextLength` passed into `AllFeatureInformation`. is this a mistake?

Reply:
Good catch! It's not in there, i'll remove it